CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ See [#296](https://github.com/latera/homs/pull/296) for more details.
 
 ### Refactoring
 - [#506](https://github.com/latera/homs/pull/506) Update ruby and js dependencies.
+- [#511](https://github.com/latera/homs/pull/511) Update ruby and js dependencies.
```

Comment:
We updated ruby version [there](https://github.com/latera/homs/pull/506).
We should not write about every updating in the changelog.